glance/files/mitaka/glance-api.conf.Debian

```diff
@@ -799,55 +799,101 @@ stores = {{ server.storage.engine }}
 # (dict value)
 #http_proxy_information =
 
+{%- if 'swift' in storage_engines %}
+
 # If True, swiftclient won't check for a valid SSL certificate when
 # authenticating. (boolean value)
+{% if server.storage.swift_store_auth_insecure is defined %}
+swift_store_auth_insecure = {{ server.storage.swift_store_auth_insecure }}
```

Comment:
BTW it's good idea to separate backends configuration into separate files and use `{% include ...` to avoid huge messy config/jinja files.